src/ci/docker/host-x86_64/disabled/dist-x86_64-dragonfly/build-toolchain.sh

```diff
@@ -14,10 +14,10 @@ echo ERROR: An error was encountered with the build.
 cat /tmp/build.log
 exit 1
 "
-  trap "$on_err" ERR
+  trap '$on_err' ERR
```

Comment:
This has different behavior, doesn't it? Currently it expands $on_err when `trap` is executed this; after it will expand $on_error when an error occurs.

Reply:
Yes, this has different behavior, however in this case it won't make a difference and it'd make ShellCheck shut up, but I can change it back if necessary.

Reply:
Please change it back, yeah. I don't change the behavior just to satisfy the linter - it's possible the script is depending on the current behavior.

Reply:
I think you missed this comment: please change `'$on_err'` back to `"$on_err"`.